vault/auth.go

```diff
@@ -569,7 +569,7 @@ func (c *Core) newCredentialBackend(ctx context.Context, entry *MountEntry, sysV
 
 	config := &logical.BackendConfig{
 		StorageView: view,
-		Logger:      c.logger,
+		Logger:      c.logger.ResetNamed(fmt.Sprintf("auth.%s", t)).With("path", entry.Path, "accessor", entry.Accessor),
```

Comment:
I admit it wasn't super clear what @evanphx meant in his last email but I _thought_ he had suggested having `auth.ldap.<accessor>` rather than having it in With. Also I think the `path` isn't important to have in every line since you can uniquely identify the path via the accessor and the lookup table.

Reply:
Yeah, i'll add the accessor to the name. However, unless we are worried about leaking mount locations, i'd vote we leave the path in. I think that's the most human friendly way of knowing which mount is outputting an error.